dali/test/python/test_operator_pad.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from nose.tools.nontrivial import raises
```

Comment:
?

Reply:
Please remove. That's probably VSCode's doing - it keeps doing it to me all the time :\

Reply:
removed